src/main/java/org/jenkinsci/test/acceptance/recorder/SupportBundle.java

```diff
@@ -0,0 +1,44 @@
+package org.jenkinsci.test.acceptance.recorder;
+
+import org.jenkinsci.test.acceptance.po.Jenkins;
+import org.jenkinsci.test.acceptance.utils.SupportBundleRequest;
+import org.junit.rules.TestWatcher;
+import org.junit.runner.Description;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class SupportBundle extends TestWatcher {
+    private static final Logger LOGGER = Logger.getLogger(SupportBundle.class.getName());
+
+    private static class SupportBundleSpec {
+        private Jenkins instance;
+        private SupportBundleRequest request;
+
+        public SupportBundleSpec(Jenkins instance, SupportBundleRequest request) {
+            this.instance = instance;
+            this.request = request;
+        }
+    }
+
+    private List<SupportBundleSpec> specs = new ArrayList<>();
+
+    public SupportBundle() {}
+
+    public void addSpec(Jenkins jenkins, SupportBundleRequest request) {
+        specs.add(new SupportBundleSpec(jenkins, request));
+    }
+
+    @Override
+    protected void failed(Throwable e, Description description) {
+        for (SupportBundleSpec spec : specs) {
```

Comment:
should this not be opt in like `RECORDER` and the others?
Just because I have a plugin installed does not mean I want these bundles does it (especially when running locally) - as bundles are not always tiny.

Reply:
How about I enable it by default in CI, and disable it locally ?

Reply:
In any case, we're mostly talking about 100kb support bundles, since they are fresh instances.

Reply:
cab03793d88d7ab843dd588811407419dd15f63d